content/applications/finance/accounting/taxes/avatax/avalara_portal.rst

```diff
@@ -0,0 +1,259 @@
+Avalara (Avatax) portal
+=======================
+
+Avalara's (*AvaTax*) management console offers account management options including: viewing/editing
+the transactions sent from Odoo to *AvaTax*, details on how the taxes are calculated, tax reporting,
+tax exemption management, and tax return resources.
+
+.. tip::
+   Avalara is the software developer of the tax software, *AvaTax*.
+
+To access the console, first, navigate to either Avalara's `sandbox
+<https://sandbox.admin.avalara.com/>`_ or `production <https://admin.avalara.com/>`_ environment.
+This will depend on which type of account was set in the :doc:`integration <../avatax>`. Log in to
+the management console.
+
+.. image:: avalara_portal/avalara_portal.png
+   :align: center
+   :alt: Avalara dashboard after logging into management portal.
+
+.. seealso::
+   For more information see Avalara's documentation: `Activate your Communications Customer Portal
+   account
+   <https://community.avalara.com/support/s/document-item?language=en_US&bundleId=qvv1656594440497&topicId=Activate_your_Communications_Customer_Portal_account.html&_LANG=enus>`_.
+
+.. _avalara_portal/transactions:
+
+Transactions
+============
+
+To access transactions, click in the :guilabel:`Transactions` link on the main dashboard upon
+logging into the :ref:`avatax_portal`. To manually access the *Transactions* page, while logged into
+the Avalara console, navigate to :menuselection:`Transactions --> Transactions`.
+
+.. image:: avalara_portal/avalara_transactions.png
+   :align: center
+   :alt: Avalara portal with the transactions shortcut highlighted.
+
+Edit transaction
+----------------
+
+Click into a transaction to reveal more details about the transaction. These details include
+sections on :guilabel:`Invoice detail`, :guilabel:`Additional info`, and :guilabel:`Customer info`.
+Click :icon:`fa-pencil` :guilabel:`Edit document details` to make changes to the transaction.
+
+A :guilabel:`Discount` can be added to adjust the invoice. This is especially useful in cases where
+the transaction has already synced with Avalara / *AvaTax*, and changes need to be made afterward.
+
+.. _avalara_portal/filter:
+
+Filter
+------
+
+Filter transactions on the :guilabel:`Transactions` page, by setting the :guilabel:`From` and
+:guilabel:`To` fields, and configuring other fields to filter by, including:
+
+ - :guilabel:`Document Status`: any of the following options, :guilabel:`All`, :guilabel:`Voided`,
+   :guilabel:`Committed`, :guilabel:`Uncommitted`, or :guilabel:`Locked`.
+ - :guilabel:`Document Code` : any of the following options, :guilabel:`Exactly match`,
+   :guilabel:`Starts with`, or :guilabel:`Contains`.
+ - :guilabel:`Customer/Vendor Code`: the customer/vendor code in Odoo (e.g. `Contact18`).
+ - :guilabel:`Country`: the country this tax was calculated in; this is a text field.
+ - :guilabel:`Region`: the region of the country, which varies based on the :guilabel:`Country`
+   selection.
```

Comment:
Scooch these bullet points back a space, no need for an indent. And remove space before colon on Document Code
```suggestion
- :guilabel:`Document Status`: any of the following options, :guilabel:`All`, :guilabel:`Voided`,
  :guilabel:`Committed`, :guilabel:`Uncommitted`, or :guilabel:`Locked`.
- :guilabel:`Document Code`: any of the following options, :guilabel:`Exactly match`,
  :guilabel:`Starts with`, or :guilabel:`Contains`.
- :guilabel:`Customer/Vendor Code`: the customer/vendor code in Odoo (e.g. `Contact18`).
- :guilabel:`Country`: the country this tax was calculated in; this is a text field.
- :guilabel:`Region`: the region of the country, which varies based on the :guilabel:`Country`
  selection.
```